README.md

```diff
@@ -34,6 +34,17 @@ cd example/
 php ../bin/stakx build
 ```
 
+### Locally Serving a Site
+
+Stakx includes a built in PHP server, which you can run with the following commands. Access your site at `http://0.0.0.0:8000/`.
+
+```bash
+cd example/
+php ../bin/stakx serve
+```
+
+Note that the `serve` command will also rebuild your site when it detects a file change, including recompiling the SASS/SCSS, if any is in use.
```

Comment:
The `serve` command will build the current page you're requesting in your browser. If there have been changes to that file, they'll be reflected next time you reload the page in your browser. It's important to specify that things are built/delivered as they're requested, the entire site isn't built during the `serve` command.

Only SCSS is supported and it's compiled whenever it's requested. So in a page with a `<link>` to a .css file that's generated by stakx, it'll recompile as needed on the next request if there's been a change. An SCSS file in the stakx world is just a static PageView.

Reply:
Thanks — change made. Let me know if it's clearer or if you think it'll need a bit more details. Also wondering if the instructions should expand on the build command. Thoughts?

Reply:
I think the README needs to be mostly rewritten so it shows you how to use the program, instead of how to build the example site, which is more development related. The README originally was for people who wanted to experiment with Stakx, now it needs to become an actual user guide.